src/MOI_wrapper.jl

```diff
@@ -180,20 +240,31 @@ function MOI.optimize!(dest::Optimizer, src::OptimizerCache)
         options[:fid] = 0
     end
 
+    realPSDdims = _map_sets(MOI.side_dimension, Ac_real, ScaledPSDCone)
+    complexPSDdims = _map_sets(MOI.side_dimension, Ac_complex, ScaledPSDCone)
     K = Cone(
-        Ac.sets.num_rows[1],
-        Ac.sets.num_rows[2] - Ac.sets.num_rows[1],
-        _map_sets(MOI.dimension, Ac, MOI.SecondOrderCone),
-        _map_sets(MOI.dimension, Ac, MOI.RotatedSecondOrderCone),
-        _map_sets(MOI.side_dimension, Ac, ScaledPSDCone),
+        Ac_real.sets.num_rows[1],
+        Ac_real.sets.num_rows[2] - Ac_real.sets.num_rows[1],
+        _map_sets(MOI.dimension, Ac_real, MOI.SecondOrderCone),
+        _map_sets(MOI.dimension, Ac_real, MOI.RotatedSecondOrderCone),
+        [realPSDdims; complexPSDdims],
+        Vector(
+            (length(
+                realPSDdims,
+            )+1):(length(realPSDdims)+length(complexPSDdims)),
+        ),
```

Comment:
Replace this `Vector()` with:
```suggestion
        convert(Vector{Int}, length(realPSDdims).+(1:length(complexPSDdims)))
```